### PR TITLE
Get price for date action in Nord Pool

### PR DESCRIPTION
**Pull request by gjohansson-ST**
## Proposed change

Adds custom action "Get price for date" to Nord Pool

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/130185
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a new custom action, "Get price for date," allowing users to retrieve past pricing information for the last two months or for tomorrow.

- **Documentation**
	- Expanded and reorganized documentation for the Nord Pool integration, including clearer instructions on using the new functionality.
	- Added a note regarding API limitations and error handling for retrieving future prices.
	- Included examples formatted in YAML for better clarity on usage.

### CODE REVIEW

File: source/_integrations/nordpool.markdown [hunk 1]
```diff
@@ -73,7 +73,42 @@ The daily average sensor is not enabled by default.
 
 The `Exchange rate` sensor is not enabled by default.
 
-## Example
+## Custom actions
```
Comment (frenck) [inline source/_integrations/nordpool.markdown]:
```suggestion
## Actions
```

File: source/_integrations/nordpool.markdown [hunk 2]
```diff
@@ -73,7 +73,42 @@ The daily average sensor is not enabled by default.
...
+
+### Get price for date
+
+As the integration entities only provides price information for the current date, use the custom action "Get price for date" to retrieve pricing information for the last two months up until tomorrow.
+
+Providing the areas and currency are optional, and will use the integration configured if omitted.
+
+See [examples](#examples) how to use in a trigger template sensor.
+
```
Comment (frenck) [inline source/_integrations/nordpool.markdown]:
```suggestion
As the integration entities only provides price information for the current date, use the action "Get price for date" to retrieve pricing information for the last two months up until tomorrow.

Providing the areas and currency are optional, and will use the integration configured if omitted.

See [examples](#examples) how to use it in a trigger template sensor.

```

File: source/_integrations/nordpool.markdown [hunk 3]
```diff
@@ -73,7 +73,42 @@ The daily average sensor is not enabled by default.
...
+{% note %}
+
+The public api only allows to see past pricing information up to 2 months.
```
Comment (frenck) [inline source/_integrations/nordpool.markdown]:
```suggestion
The public API only allows us to see past pricing information for up to 2 months.
```

File: source/_integrations/nordpool.markdown [hunk 4]
```diff
@@ -73,7 +73,42 @@ The daily average sensor is not enabled by default.
...
+
+Tomorrow's prices are typically released around 13:00 CET, and therefore trying to get tomorrow's prices before that time will generate an error which needs to be considered in such use-case.
```
Comment (frenck) [inline source/_integrations/nordpool.markdown]:
```suggestion
Tomorrow's prices are typically released around 13:00 CET, and trying to get them before that time will generate an error that needs to be considered in such a case.
```

File: source/_integrations/nordpool.markdown [hunk 5]
```diff
@@ -93,3 +128,36 @@ template:
...
+        unique_id: se3_tomorrow_low_price
+        state: >
+          {% set data = namespace(prices=[]) %}
+          {% for state in tomorrow_price['SE3'] %}
+            {% set data.prices = data.prices + [state.price] %}
+          {% endfor %}
+          {{min(data.prices)}}
+        attributes:
+          data: "{{ tomorrow_price['SE3'] }}"
+```
```
[the remark this answers is not in the document]
Comment (frenck) [reply source/_integrations/nordpool.markdown]: Nice catch!

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by gjohansson-ST** [comment]
@coderabbitai review

**Review by frenck — APPROVED** [review]
Thanks, @gjohansson-ST 👍 

../Frenck

**Pull request merged**